src/ActionMonitor/Monitors/UserMonitor.php

```diff
@@ -13,12 +13,42 @@ class UserMonitor extends Monitor {
 	 */
 	protected $users_before_delete;
 
+	/**
+	 * IDs of posts to reassign
+	 * @var array
+	 */
+	protected $post_ids_to_reassign;
+
 	/**
 	 * Initialize UserMonitor Actions
 	 *
 	 * @return mixed|void
 	 */
-	public function init() {
+	public function init() {if ( ! empty( $post_ids ) && is_array( $post_ids ) )  {
```

Comment:
Looks like the formatting is a bit weird here. And where does $post_ids come from? Is that a global?

Reply:
😱  this was some funky copy-pasta.